shopinvader_api_sale/schemas.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Akretion (http://www.akretion.com).
+# @author Florian Mounier <florian.mounier@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from typing import Annotated
+
+from extendable_pydantic import StrictExtendableBaseModel
+from pydantic import Field
+
+from odoo import api
+
+from odoo.addons.shopinvader_schema_sale.schemas.sale_line import SaleLine
+
+
+class SaleLineWithSale(SaleLine):
+    order_id: int
```

Comment:
maybe it will be better to return an object 
"order": SaleOrderFromLine 

So at least we can return inside the "id", "name", "state", "date_order" ?

Reply:
Alright as you wish